sqltocsv.go

```diff
@@ -45,13 +45,10 @@ func main() {
 	}
 
 	csvWriter := csv.NewWriter(os.Stdout)
-	// If I ever need to support more than tabs/commas, this needs improving
-	if cfg.delimeter != "comma" {
-		if comma, ok := delimeters[cfg.delimeter]; ok {
-			csvWriter.Comma = comma
-		} else {
-			log.Printf("Warning: No known delimeter for %s, defaulting to Comma", cfg.delimeter)
-		}
+	if comma, ok := delimeters[cfg.delimeter]; ok {
```

Comment:
Spelling error :)

Makem all the time

On Aug 21, 2016 11:24 PM, "hydroflame" <notifications@github.com> wrote:

> In sqltocsv.go
> <https://github.com/StabbyCutyou/sqltocsv/pull/2#discussion_r75614014>:
>
> > @@ -45,13 +45,10 @@ func main() {
> >  	}
> >
> >  	csvWriter := csv.NewWriter(os.Stdout)
> > -	// If I ever need to support more than tabs/commas, this needs improving
> > -	if cfg.delimeter != "comma" {
> > -		if comma, ok := delimeters[cfg.delimeter]; ok {
> > -			csvWriter.Comma = comma
> > -		} else {
> > -			log.Printf("Warning: No known delimeter for %s, defaulting to Comma", cfg.delimeter)
> > -		}
> > +	if comma, ok := delimeters[cfg.delimeter]; ok {
>
> this is a misnomer, should be called like "separator" or "delimiter"
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/StabbyCutyou/sqltocsv/pull/2/files/2bbc08f188d504576a64a14140fa56102449d13b..003c2bb0891131ca1675205e7eb2d502f4923679#r75614014>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABdTKrl3DbmEekt4JHyxY_TJWLWUNkP_ks5qiRZQgaJpZM4JljSO>
> .
>